lib/src/indentation.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:math';
+
+const _defaultIndent = '    ';
+const _maxLinesToCheck = 10;
```

Comment:
@Chinmay-KB Note that I have added a limit otherwise it would iterate the whole file if no indentation is present.